### ISSUE TITLE
How to set user/password to none?

### ISSUE THREAD

**Issue by Cantello** [djmaze/tiddlywiki-docker#18]
I'm using this TW as a collaborative notebook for a larger group and would like to not use username/password. Would this be possible?

**Comment by djmaze** [comment]
Yes, it is possible, see this commit: https://github.com/djmaze/tiddlywiki-docker/commit/1d356df7010758c79f67254ba3be5d35a39765c4

Take care though, I don't think TW can be used simultaneously by several people. It is not built for that, AFAIK.

**Comment by LinuCC** [comment]
@djmaze would it be possible to push that fix as a new version back to the Docker Hub?

**Comment by djmaze** [comment]
Oh, right, I just triggered a new build on the hub. It should now contain all the fixes.

Please note that I am not developing this any further. Since [someone said he will be updating his fork regularly](https://github.com/djmaze/tiddlywiki-docker/issues/19#issuecomment-681862396) I will probably be pointing to that soon.
